From 592837abc8b07f9b28898fa91c31658e2beecfc4 Mon Sep 17 00:00:00 2001
From: Ronny Berndt <ronny@apache.org>
Date: Sat, 11 Jun 2022 14:18:54 +0200
Subject: [PATCH] Add Erlang 25 support

Before merging this PR, PR #16 in couchdb-snappy needs to get merged and tagged with CouchDB-1.0.8
---
 rebar.config.script | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rebar.config.script b/rebar.config.script
index 31855098b3..44c4d61b1a 100644
--- a/rebar.config.script
+++ b/rebar.config.script
@@ -186,7 +186,7 @@
 AddConfig = [
     {cover_enabled, true},
     {cover_print_enabled, true},
-    {require_otp_vsn, "23|24|25"},
+    {require_otp_vsn, "23|24|25|26"},
     {deps_dir, "src"},
     {deps, lists:map(MakeDep, DepDescs ++ OptionalDeps)},
     {sub_dirs, SubDirs},
--- apache-couchdb-3.3.2/src/snappy/rebar.config	2023-06-27 16:23:39.000000000 +0200
+++ apache-couchdb-3.3.2/src/snappy/rebar.config.new	2023-06-27 16:35:58.371758506 +0200
@@ -1,4 +1,4 @@
-{require_otp_vsn, "R14|R15|R16|17|18|19|20|21|22|23|24|25"}.
+{require_otp_vsn, "R14|R15|R16|17|18|19|20|21|22|23|24|25|26"}.
 
 {erl_opts, [debug_info, warn_unused_vars, warn_shadow_vars, warn_unused_import]}.
 {port_sources, ["c_src/*.cc", 
